modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogKeyboardNavTest.ts

```diff
@@ -21,11 +22,13 @@ describe('browser.tinymce.plugins.table.TableDialogKeyboardNavTest', () => {
   const pressTabKey = (editor: Editor) => TinyUiActions.keydown(editor, Keys.tab());
 
   // Assert focus is on the expected form element
-  const pAssertFocusOnItem = (label: string, selector: string) => FocusTools.pTryOnSelector(
-    `Focus should be on: ${label}`,
-    SugarDocument.getDocument(),
-    selector
-  );
+  const pAssertFocusOnItem = (label: string, selectorOrElement: string | SugarElement<Node>) => {
+    if (typeof selectorOrElement === 'string') {
```

Comment:
Is it so hard to add a second assert method for the 7 cases that need it? Both would be a single line 🤷